src/data/ACTIONS/MRD.js

```diff
@@ -7,56 +7,92 @@ export default {
 		id: 31,
 		name: 'Heavy Swing',
 		icon: 'https://xivapi.com/i/000000/000260.png',
+		potency: 160,
 		onGcd: true,
+		combo: {
+			start: true,
+		},
 	},
 
 	SKULL_SUNDER: {
 		id: 35,
 		name: 'Skull Sunder',
 		icon: 'https://xivapi.com/i/000000/000257.png',
 		onGcd: true,
+		potency: 100,
+		combo: {
+			from: 31,
```

Comment:
This is probably a bad idea, but here's a way to avoid having the `id` magic number repeated in more than one place:

```ts
export { actions as default }
const actions = {
  HEAVY_SWING: {
    ...
  },

  SKULL_SUNDER: {
    ...
    combo: {
      get from () { return actions.HEAVY_SWING }
      ...
    }
  }
}
```

Reply:
Alternative:

```ts
const HEAVY_SWING = {
  ...
}

const SKULL_SUNDER = {
  ...
  combo: {
    from: HEAVY_SWING.id
    ...
  }
}

export default { HEAVY_SWING, SKULL_SUNDER, ... }
```

Reply:
Including combo data in the ACTIONS files is the current precedent, as seen here:
https://github.com/xivanalysis/xivanalysis/blob/master/src/data/ACTIONS/ROG.js#L24
https://github.com/xivanalysis/xivanalysis/blob/master/src/data/ACTIONS/RDM.js#L86
https://github.com/xivanalysis/xivanalysis/blob/master/src/data/ACTIONS/LNC.js#L23

While I generally agree with avoiding duplicate magic numbers, I feel like it's acceptable in this specific use case, especially since we're eventually hoping to automate building these files and combo data should come along with that.